linux/ibus-keyman/tests/dbus-test-server/KmDbusTestServer.cpp

```diff
@@ -98,11 +98,9 @@ void KmDbusTestServer::Loop()
     return;
   }
 
-  guint exitFlag = FALSE;
-
   // Install the object
   ret = sd_bus_add_object_vtable(bus, NULL, KEYMAN_TESTSVC_OBJECT_PATH,
-    KEYMAN_TESTSVC_INTERFACE_NAME, test_service_vtable, &exitFlag);
+    KEYMAN_TESTSVC_INTERFACE_NAME, test_service_vtable, this);
```

Comment:
So this the function definition has the last argument as void*,. Where does is the call back that asigns the value to this->exitFlag?